data/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/Conversation.kt

```diff
@@ -299,7 +299,7 @@ sealed class ConversationDetails(open val conversation: Conversation) {
         override val conversation: Conversation,
         val hasOngoingCall: Boolean = false,
         val isSelfUserMember: Boolean,
-        val isSelfUserCreator: Boolean,
+        val selfUserTeamId: TeamId?,
```

Comment:
There is no need to add the user self team ID to the conversation model, since it is fixed and is not really conversation related, if you need it in the UI part to get whether a conversation can be deleted or not, please expose it as its own use case and use it in the UI